### ISSUE TITLE
Admin does not refresh immediately after backup completes

### ISSUE THREAD

**Issue by pdewouters** [humanmade/backupwordpress#839]
This "issue" ( perception of slowness ) was introduced after introducing the use of Backdrop for running a backup. It's probably due to the fact that it's waiting on the next heartbeat pulse.
To reproduce, start a small DB backup, and monitor the backups folder, you'll notice that the backup completes very fast, but there is a delay in the admin to reload and complete.

**Comment by willmot** [comment]
Perhaps using Heartbeat doesn't actually make sense for us, we really need something more predictable.

**Comment by pdewouters** [comment]
Yes, this could tie in with the loopback/cron issue, if we switch to a custom JS polling implementation?

**Comment by willmot** [comment]
I'm definitely leaning towards switching away from Backdrop & Heartbeat for manual backups, they seem to have been the source of a lot of issues and haven't really brought us any benefits (other than reducing the amount of first-party code).

I would probably continue to use Backdrop for the site size calculation as that's not a critical core feature.

**Comment by pdewouters** [comment]
@willmot should we have a separate ticket for discussing the core implementation for backups, both manual and scheduled? Seem it is fragmented across other issues:
https://github.com/humanmade/backupwordpress/issues/752#issuecomment-122998982
https://github.com/humanmade/backupwordpress/issues/821

**Comment by willmot** [comment]
@pdewouters yep sounds sensible, could you create a new issue that summarises where we are at with both manual and automatic backups (don't think we should separate them out until we're sure they warrant different solutions) and what the various options / next steps are. We can then continue discussion and move towards solutions. 

All these other issues can then be referenced from that new ticket and closed as dupes.

**Comment by pdewouters** [comment]
Fixed by https://github.com/humanmade/backupwordpress/pull/937
